analyses/molecular-subtyping-chordoma/00-subset-files-for-chordoma.R

```diff
@@ -0,0 +1,79 @@
+# This script subsets the focal copy number, RNA expression, histologies` 
+# files to include only Chordoma samples.
+
+# Written originally Chante Bethell 2019 
+# (Adapted for this module by Candace Savonen 2020)
+#
+# #### USAGE
+# This script is intended to be run via the command line from the top directory
+# of the repository as follows:
+#
+# Rscript 'analyses/molecular-subtyping-chordoma/00-subset-files-for-chordoma.R'
+
+
+#### Set Up --------------------------------------------------------------------
+
+# Get `magrittr` pipe
+`%>%` <- dplyr::`%>%`
+
+#### Directories and Files -----------------------------------------------------
+
+# Detect the ".git" folder -- this will in the project root directory.
+# Use this as the root directory to ensure proper sourcing of functions no
+# matter where this is called from
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+# Set path to results and plots directory
+results_dir <-
+  file.path(root_dir, "analyses", "molecular-subtyping-chordoma", "chordoma-subset")
+
+if (!dir.exists(results_dir)) {
+  dir.create(results_dir)
+}
+
+scratch_dir <- file.path(root_dir, "scratch")
```

Comment:
Suggestion: create a subdirectory in `scratch` for this analysis to avoid collisions.

Reply:
This is leftover from ATRT, I don't use the scratch at all. Gonna delete this line.